h2o-core/src/test/java/water/parser/ParseFolderTestBig.java

```diff
@@ -50,10 +52,38 @@ public void testKDDCup() {
     }
   }
 
+  @Test
+  public void testPUBDEV4026() {
+    Scope.enter();
+    String fname = "bigdata/PUBDEV4026.svm";
+    try {
+      File f = FileUtils.getFile(fname);
+      NFSFileVec nfs = NFSFileVec.make(f);
+      final ParseSetup globalSetup = ParseSetup.guessSetup(new Key[]{nfs._key}, false, ParseSetup.GUESS_HEADER);
+      ParseWriter.ParseErr[] errors = globalSetup._errs;
+      
+      assertTrue("Got errors: " + Arrays.toString(errors), errors.length == 0);
+      Job<Frame> job = ParseDataset.parse(Key.make("PUBDEV4026.hex"), new Key[]{nfs._key}, true, globalSetup, false)._job;
+      while (job.progress() < 1.0) {
+        System.out.print(((int) (job.progress() * 1000.0)) / 10.0 + "% ");
+        try {
+          Thread.sleep(1000);
+        } catch (InterruptedException ignore) { /*comment to disable ideaJ warning*/}
+      }
```

Comment:
Yep, you are right - I was too quick with sending the comment (updated it in the meantime)